src/server/acl/acl_family.cc

```diff
@@ -990,6 +990,22 @@ using OptCat = std::optional<uint32_t>;
 // bool == true if +
 // bool == false if -
 std::pair<OptCat, bool> AclFamily::MaybeParseAclCategory(std::string_view command) const {
+  if (absl::EqualsIgnoreCase(command, "ALLCOMMANDS")) {
+    auto res = cat_table_.find("ALL");
```

Comment:
`ALL` will always exist in `cat_table_` so you don't really need to call `find` and compare that `cat_table_.end()`. 

You can just use `cat_table_["ALL"]` :)

Reply:
Yeah, you are right about comparing to ```cat_table_.end()```

I tried to used operator[] but it was failing due to type mismatch and even after casting to ```std::string_view``` it was failing due to -fpermissive (this arguement discards qualifier)  so finally used ```.at()```